### PR TITLE
Feat/1269 lnurlauth popup

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

lnurlAuth support via the "Send" within the popup

### Link this PR to an issue

#1269 

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes

none

### How has this been tested?

manually with fiatjefs playground and stacker.news

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/extension/background-script/actions/ln/auth.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { MessageLnurlAuth } from "~/types";
```
Comment (bumi) [inline src/extension/background-script/actions/ln/auth.ts:1]: the `ln` folder is used for actions that call LN functions on the connector. 
so this does not fit in here. we have a lnurl folder, don't we?

Comment (escapedcat) [reply src/extension/background-script/actions/ln/auth.ts:1]: Yes, will move. We talked about it but I wasn't sure.

Comment (escapedcat) [reply src/extension/background-script/actions/ln/auth.ts]: Have a look again

File: src/extension/background-script/actions/lnurl/lnurlAuth.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { MessageLnurlAuth } from "~/types";
+
+import { auth as lnurlAuth } from "../lnurl/auth";
+
+const auth = async (message: MessageLnurlAuth) => {
```
Comment (bumi) [inline src/extension/background-script/actions/lnurl/lnurlAuth.ts:5]: some tricky naming here. `auth as lnurlAuth` vs. the new `auth` here

**Review by bumi — APPROVED** [review]
utACK. I will test more of the whole LNURL* options in master

**Pull request merged**